signature/jws/envelope.go

```diff
@@ -129,12 +124,29 @@ func (e *envelope) Payload() (*signature.Payload, error) {
 	if e.internalEnvelope == nil {
 		return nil, &signature.MalformedSignatureError{Msg: "missing jws signature envelope"}
 	}
+	// parse protected header to get payload context type
 	protected, err := parseProtectedHeaders(e.internalEnvelope.Protected)
 	if err != nil {
 		return nil, err
 	}
+
+	// convert JWS to JWT
+	tokenString := compactJWS(e.internalEnvelope)
+
+	// parse JWT to get payload context
+	parser := jwt.NewParser(
+		jwt.WithValidMethods([]string{"PS256", "PS384", "PS512", "ES256", "ES384", "ES512"}),
```

Comment:
```suggestion
               var validMethods = []string{"PS256", "PS384", "PS512", "ES256", "ES384", "ES512"}
		jwt.WithValidMethods(validMethods),
```
Can we use a `validMethod` slice instead of hard code string in the code

Reply:
Good idea. Updated.